src/wp-includes/class-wp-autoload.php

```diff
@@ -0,0 +1,502 @@
+		'wp_script_modules'                           => 'wp-includes/class-wp-script-modules.php',
+		'wp_scripts'                                  => 'wp-includes/class-wp-scripts.php',
+		'wp_session_tokens'                           => 'wp-includes/class-wp-session-tokens.php',
+		'wp_simplepie_file'                           => 'wp-includes/class-wp-simplepie-file.php',
+		'wp_simplepie_sanitize_kses'                  => 'wp-includes/class-wp-simplepie-sanitize-kses.php',
+		'wp_site_query'                               => 'wp-includes/class-wp-site-query.php',
+		'wp_site'                                     => 'wp-includes/class-wp-site.php',
+		'wp_styles'                                   => 'wp-includes/class-wp-styles.php',
+		'wp_tax_query'                                => 'wp-includes/class-wp-tax-query.php',
+		'wp_taxonomy'                                 => 'wp-includes/class-wp-taxonomy.php',
+		'wp_term_query'                               => 'wp-includes/class-wp-term-query.php',
+		'wp_term'                                     => 'wp-includes/class-wp-term.php',
+		'wp_text_diff_renderer_inline'                => 'wp-includes/class-wp-text-diff-renderer-inline.php',
+		'wp_text_diff_renderer_table'                 => 'wp-includes/class-wp-text-diff-renderer-table.php',
+		'wp_textdomain_registry'                      => 'wp-includes/class-wp-textdomain-registry.php',
+		'wp_theme_json_data'                          => 'wp-includes/class-wp-theme-json-data.php',
+		'wp_theme_json_resolver'                      => 'wp-includes/class-wp-theme-json-resolver.php',
+		'wp_theme_json_schema'                        => 'wp-includes/class-wp-theme-json-schema.php',
+		'wp_theme_json'                               => 'wp-includes/class-wp-theme-json.php',
+		'wp_theme'                                    => 'wp-includes/class-wp-theme.php',
+		'wp_token_map'                                => 'wp-includes/class-wp-token-map.php',
+		'wp_user_meta_session_tokens'                 => 'wp-includes/class-wp-user-meta-session-tokens.php',
+		'wp_user_query'                               => 'wp-includes/class-wp-user-query.php',
+		'wp_user_request'                             => 'wp-includes/class-wp-user-request.php',
+		'wp_user'                                     => 'wp-includes/class-wp-user.php',
+		'walker'                                      => 'wp-includes/class-wp-walker.php',
+		'wp_widget_factory'                           => 'wp-includes/class-wp-widget-factory.php',
+		'wp_widget'                                   => 'wp-includes/class-wp-widget.php',
+		'wp_xmlrpc_server'                            => 'wp-includes/class-wp-xmlrpc-server.php',
+		'wp'                                          => 'wp-includes/class-wp.php',
+		'wpdb'                                        => 'wp-includes/class-wpdb.php', // Defines some constants.
+
+		/* Classes in the wp-includes/fonts folder. */
```

Comment:
```suggestion
		// Classes in the wp-includes/fonts folder.
```

Single line comment style per coding standards

:1234: Applies throughout this file.